### DISCUSSION TITLE
Allow to set retry count for xcode_install to resume failed downloads (xcpretty/xcode-install finally supports that)

### DISCUSSION THREAD

**Discussion by sergebat** [fastlane/fastlane#17608] [Ideas]
Even with reasonably good internet connection xcode_install rarely succeed to download entire 9 GB XCode bundle, and fails with "curl: (18) transfer closed with 4099344615 bytes remaining to read"

It does attempt to resume download, but only 3 times. 

xcpretty/xcode-install finally made retry count it configurable.  https://github.com/xcpretty/xcode-install/pull/400

Is there a way to pass that parameter from the fastlane? I don't see it anywhere in the code or doc, but maybe I am missing something?

If not, can someone with better understanding of how this action works https://github.com/fastlane/fastlane/blob/af2d0364073ac3c517aacf840b75b53847241f7f/fastlane/lib/fastlane/actions/xcode_install.rb consider adding this parameter? This will be most appreciated!

**Comment by rogerluan** [comment]
Hey @sergebat 👋 

Great idea! I'll open a PR 💪

**Reply by rogerluan** [reply]
Here it is 🚀 https://github.com/fastlane/fastlane/pull/17691
